test/test_train_mp_imagenet.py

```diff
@@ -38,7 +38,25 @@
     },
     '--profile': {
         'action': 'store_true',
-    }
+    },
+    '--persistent_workers': {
+        'action': 'store_true',
+    },
+    '--prefetch_factor': {
+        'type': int,
+    },
+    '--loader_prefetch_size': {
+        'type': int,
+    },
+    '--device_prefetch_size': {
+        'type': int,
+    },
+    '--host_to_device_transfer_threads': {
+        'type': int,
+    },
+    '--use_optimized_kwargs': {
+        'action': 'store_true',
```

Comment:
can we make a `str`? so user can use it like `--use_optimized_kwargs tpuv4` and verifiy if it works?

Reply:
Yeah I can do that.